src/main/java/org/yu55/yagga/handler/git/GitPullScheduler.java

```diff
@@ -0,0 +1,26 @@
+package org.yu55.yagga.handler.git;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GitPullScheduler {
+
+    private static final Logger logger = LoggerFactory.getLogger(GitPullScheduler.class);
+
+    private GitRepositories gitRepositories;
+
+    @Autowired
+    public GitPullScheduler(GitRepositories gitRepositories) {
+        this.gitRepositories = gitRepositories;
+    }
+
+    @Scheduled(fixedRate = 1000 * 60)
```

Comment:
Please consider making fixedRate configurable in yagga-config.properties.
To chieve this instead of fixedRate annotation parameter You should probably use fixedRateString (as described here: http://stackoverflow.com/a/16463403/5533288)